Running-Mastodon/Production-guide.md

```diff
@@ -111,7 +111,14 @@ server {
 
 ## Running in production without Docker
 
-It is recommended to create a special user for mastodon on the server (you could call the user `mastodon`), though remember to disable outside login for it. You should only be able to get into that user through `sudo -u mastodon`.
+It is recommended to create a special user for mastodon on the server (you could call the user `mastodon`), though remember to disable outside login for it. You should only be able to get into that user through `sudo -Hsu mastodon`.
+
+This command will create the user as needed:
+
+    sudo adduser --system --group --disabled-login --gecos "" --home /home/mastodon mastodon
```

Comment:
`adduser` isn't a unix standard, it's only available on debian-based distributions. To make this more portable, we could use `useradd` which is standard. The equivalent useradd command is `sudo useradd --system --user-group --shell /bin/false --create-home --home /home/mastodon mastodon`